bundler/helpers/Gemfile.lock

```diff
@@ -0,0 +1,9 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+
+PLATFORMS
+  ruby
+  x86_64-darwin-19
+
+DEPENDENCIES
```

Comment:
This is funky, we need to commit this without a `BUNDLED WITH` as it's loaded from setting `BUNDLE_GEMFILE` to the above `Gemfile` the `when running `bundle exec` on customers manifests, if we don't set this bundler complains that the gems defined in the customers gemfile are not installed.

Similarly, `require "file"` used to happen here https://github.com/dependabot/dependabot-core/blob/b53df52e75d5eb1c585d6339b90f03e676e5ca73/common/dependabot-common.gemspec#L3 so worked if `File` was used in a customers `.gemspec` but not required 😑